ledger/src/blockstore_processor.rs

```diff
@@ -65,7 +65,7 @@ use {
 };
 
 // it tracks the block cost available capacity - number of compute-units allowed
-// by max block cost limit
+// by max block cost limit, plus a 10% buffer by default
```

Comment:
the 10% buffer seems to be arbitrary. In a convo with Stephen the other day, we agree it might be better to not have buffer at Replay Stage, but stick to the const limits. Should needs of buffer rises, it could be implemented on leader side.

Reply:
Okay, though we DO need to tweak the logic a little bit so that we don't fail if we go over the limit in the very last batch, because at that point we're done, and that's a legal state.

Reply:
I'm replacing it with a 0.1% buffer. Unless my understanding is incorrect, we need a small buffer (could be 0.01%, or even something like "1000", but more than 0) so that if the leader has a valid mis-estimation of the bpf cost of the final transaction, which puts the total cost of the block at just slightly higher than the cap, we don't end up completely throwing it out across the whole network. (even worse.. doing all the work, and THEN throwing the block out)

Reply:
leader use actual bpf cost to update cost_tracker, which should be the exact same number replay would check out. So there is really no need of "buffer".

Reply:
Okay, if it's not possible for the leader to mis-estimate, compute a final transaction on a block, and discover that it's now actually at 48,000,100 cu for the block (but that transaction still gets included), then we don't need any buffer.